.vscode/extensions.json

```diff
@@ -2,8 +2,6 @@
   "recommendations": [
     "dbaeumer.vscode-eslint",
     "esbenp.prettier-vscode",
-    "msjsdiag.debugger-for-chrome",
-    "eg2.vscode-npm-script",
     "jpoissonnier.vscode-styled-components",
```

Comment:
This one is deprecated as well 
https://github.com/styled-components/vscode-styled-components

`The jpoissonnier.vscode-styled-components version will recieve no more updates.`. It was removed from marketplace.

Reply:
Oh, removed :/ maybe that's why vscode didn't warn me about it.

Replacing with styled-components.vscode-styled-components